    public void valid1()
        throws RemoteException;

